### PR TITLE
Preservation import

### PR DESCRIPTION
**Pull request by jolf**
Support of the ImportFromPreservation workflow, which is described here:
https://github.com/Det-Kongelige-Bibliotek/Yggdrasil/wiki/Import-from-preservation

### CODE REVIEW

File: app/helpers/mq_listener_helper.rb [hunk 1]
```diff
@@ -18,6 +18,22 @@ def handle_preservation_response(message)
     update_preservation_metadata_for_element(message, element)
   end
 
+  def handle_preservation_import_response(message)
```
Comment (jolf) [inline app/helpers/mq_listener_helper.rb]: missing method description

File: app/controllers/content_files_controller.rb [hunk 2]
```diff
@@ -66,4 +66,16 @@ def update
   def set_file
     @file = ContentFile.find(URI.unescape(params[:id]))
   end
+
+  def initiate_import_from_preservation
+    if @file.send_request_to_import(params['type'])
+      flash[:notice] = t('content_file.flashmessage.initiated_import_from_preservation')
+      # It only creates a new job, if no such job already exists.
+      ReceiveResponsesFromPreservationJob.schedule_new_job
```
Comment (davidgrove73) [inline app/controllers/content_files_controller.rb]: We should consider using Resque-scheduler to schedule regular jobs, listening for preservation responses

File: app/controllers/view_file_controller.rb [hunk 3]
```diff
@@ -18,4 +19,73 @@ def show
...
+      flash[:error] = t('flashmessage.standard_error')
+      logger.error standard_error.to_s
+      render status: 400, nothing: true
+    end
+  end
+
+  private
+  # Handle the import of the preservation import HTTP POST
+  def handle_preservation_import(params)
+    # only support content of ContentFile import
```
Comment (davidgrove73) [inline app/controllers/view_file_controller.rb]: Usually this type of functionallity is placed in the model. As an method on the ContentFile object.

**Comment by davidgrove73** [comment]
:+1:

File: app/controllers/view_file_controller.rb (hunk 3 above)
Comment (jolf) [inline app/controllers/view_file_controller.rb]: Ok, I'll do that.

**Pull request merged**